src/Home.js

```diff
@@ -64,14 +77,49 @@ class Home extends Component {
       return 'loading...';
     }
 
+    const definitionMatch = R.compose(
+      R.any(R.contains(this.state.search)),
```

Comment:
Do we need to use `any` here? Won't `chain` return a single array?

Reply:
Nevermind. I see why now, but should we add a comment so we remember why we need this later?

Reply:
You have a comment in mind? 😄